.github/pull_request_template.md

```diff
@@ -0,0 +1,53 @@
+<!-- DO NOT REMOVE BEGIN -->
+[style guide]: https://unity-docs.gitbook.io/style-guide/
+[release note guide lines]: https://unity-docs.gitbook.io/style-guide/style/release-notes
+[coding conventions]: https://github.cds.internal.unity3d.com/unity/unity-meta/blob/stable/ReferenceSource/CSharp/Assets/CSharpReference.cs
+<!-- DO NOT REMOVE END -->
+
+### Proposed change(s)
+
+- Describe the changes made in this PR.
+
+### Platform and Editor version used to develop and test this PR
+- e.g. Windows, 2019.4.11f1
+
+### Related links
+- Trello/JIRA/Airtable/FogBugz
+
+### Affected sample(s)
+
+- [ ] 2DSpaceShooter
+- [ ] Invaders
+- [ ] Client Driven
+- [ ] API Diorama
```

Comment:
Extremely minor and personal preference but I'd suggest calling the project "API Showcase" or "API Demo" instead of "Diorama" since "Showcase" or "Demo" sounds more common and obvious, especially in devsphere :)